CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ## x.y.z
 
+### Added
+
+### Changed
+
+### Deprecated
+- Support for PHP 7.1 will be removed in Emogrifier 6.0.
+
+### Removed
+
+### Fixed
+
+## 4.0.0
+
```

Comment:
Should there be an entry for 3.1.0 as well?  We've previously had entries for e.g. 2.1.0, 2.1.1 and 2.2.0.

Reply:
2.1.0 etc. ended up in the 3.0.0 changelog because we released the 2.1.0 versions etc. from master (which then became version 3.0.0), whereas 3.1.0 comes from a branch off the 3.0.0 tag because we already had some breaking changes in master at that time (if I remember correctly).

To make things more straightforward (so people can see the changes for 4.0.0 over 3.1.0), I'll update the changelog accordingly.

Reply:
Done and pushed.